import-cluster/hub.sh

```diff
@@ -30,6 +30,13 @@ fi
 PARAMS="$(applier -d params.yaml $IN -o /dev/stdout -s)"
 NAME=$(echo "$PARAMS" | grep "name:" | cut -d ":" -f2 | sed 's/^ //')
 KBCG=$(echo "$PARAMS" | grep "kubeConfig:" | cut -d ":" -f2 | sed 's/^ //')
+TOKEN=$(echo "$PARAMS" | grep "token:" | cut -d ":" -f2 | sed 's/^ //')
+VERSION=$(oc get multiclusterhubs.operator.open-cluster-management.io multiclusterhub -o=jsonpath='{.status.currentVersion}{"\n"}')
+if [ "$VERSION" \< "2.3.0" ] && ([ "$KBCG" != "" ] || [ "$TOKEN" != "" ])
```

Comment:
will this compare the version?

Reply:
yes, It takes times to me to find how to compare. I tried with "2..4.0" and it works ;-)